extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,69 @@
+import  Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+import StringBuilder from "ad-engine/src/utils/string-builder";
+
+Context.extend(config);
+let supportedTemplates = [ BigFancyAdBelow ];
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+function buildVastAdUnit(slotName) {
+	return StringBuilder.build(
+		Context.get(`vast.adUnitId`),
+		Object.assign({}, Context.get('targeting'), Context.get(`slots.${slotName}`))
+	);
+}
+
+function init(slotRegistry) {
+	SlotService.getBySlotName = (id) => {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Convert this ES6 arrow function into a regular one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AArrowFunctions)

[project-id]: wikia:app